D/Deno/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Deno"
+version = v"1.9.2"
+
+release_url = "https://github.com/denoland/deno/releases/download/v$version"
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("$release_url/deno-x86_64-unknown-linux-gnu.zip", "05c69483ff0b98c7b26666e35672b71f3dcde1df523db10ccee60510275be1c5"; unpack_target = "x86_64-linux-cxx11"),
```

Comment:
Ugh, sorry, my mistake
```suggestion
    ArchiveSource("$release_url/deno-x86_64-unknown-linux-gnu.zip", "05c69483ff0b98c7b26666e35672b71f3dcde1df523db10ccee60510275be1c5"; unpack_target = "x86_64-linux-gnu"),
```